src/main/java/core/basesyntax/Triangle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Triangle extends Shape {
+    private final double leg1;
```

Comment:
let's add setters and getters so user will be able to modify shape configuration if necessary